core/src/main/java/org/bitcoinj/testing/FakeAltNetParams.java

```diff
@@ -0,0 +1,75 @@
+package org.bitcoinj.testing;
+
+import org.bitcoinj.base.Coin;
+import org.bitcoinj.base.utils.MonetaryFormat;
+import org.bitcoinj.core.BitcoinSerializer;
+import org.bitcoinj.core.Block;
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.core.StoredBlock;
+import org.bitcoinj.core.VerificationException;
+import org.bitcoinj.store.BlockStore;
+import org.bitcoinj.store.BlockStoreException;
+
+/**
+ * Fake Alt-net subclass of {@link NetworkParameters} for unit tests.
+ */
+public class FakeAltNetParams extends NetworkParameters {
```

Comment:
I wonder if "Mock" would perhaps be a better prefix for these two classes, rather than "Fake"?

Reply:
Changed to "Mock" in latest rebase/push.